src/components/User/service.ts

```diff
@@ -7,7 +7,7 @@ const UserModel = require('./model');
  * @summary get list of all users
  * @returns Promise<UserModel[]>
  */
-function findAll() {
+function findAll(): Promise < object[] > {
```

Comment:
Promise < object[] >  that is not correct type

must be like Promise<UserDocument[]>